docs/development_guide.md

```diff
@@ -88,6 +142,15 @@ For other projects you can use the `--project` and `--gs_dir` options of the
 script.
 
 ### Pushing changes to your fork's branch
+
+Before pushing changes, make sure the pylint checks pass. To install pylint:
+```bash
+sudo apt-get install pylint
```

Comment:
Did you successfully use the pylint that comes with your linux distribution? I had problems with mine originally (Trusty at the time) and I tried again on Rodete but it gives me a lot of warnings (it is an older version). I use a  local pylint in my venv: `pip install --upgrade pylint`

Reply:
Yes. I tried pip install --upgrade pylint in the first place and somehow it did not work.

Reply:
So please update as we discussed.

Reply:
Done.